plugins/actix-web/src/lib.rs

```diff
@@ -314,3 +351,40 @@ impl actix_web::dev::Factory<(), Ready<Result<HttpResponse, Error>>, Result<Http
         fut_ok(HttpResponse::Ok().json(&*self.0.read()))
     }
 }
+
+#[cfg(feature = "swagger-ui")]
+#[derive(Clone)]
+struct SwaggerUIHandler(String);
+
+#[cfg(feature = "swagger-ui")]
+#[derive(Serialize)]
+struct SwaggerUIContext {
+    api_spec_url: String,
+}
+
+#[cfg(feature = "swagger-ui")]
+static TEMPLATE: &str = include_str!("../templates/swagger-ui.html");
```

Comment:
Let's move this from `plugins/actix-web/templates` to  `plugins/templates` so that it's applicable to other places as well.